ktor-server/ktor-server-plugins/ktor-server-body-limit/jvmAndNix/src/io/ktor/server/plugins/bodylimit/RequestBodyLimit.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2014-2023 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.plugins.bodylimit
+
+import io.ktor.server.application.*
+import io.ktor.server.plugins.*
+import io.ktor.server.request.*
+import io.ktor.utils.io.*
+import io.ktor.utils.io.pool.*
+import kotlinx.coroutines.*
+
+/**
+ * A configuration for [RequestBodyLimit] plugin.
+ */
+public class RequestBodyLimitConfig {
```

Comment:
Could you tell me if we can have a method on `call.request` to adjust requirements for a specific call in routing?